tests/integration_tests/scheduler/test_lsf_driver.py

```diff
@@ -100,3 +118,12 @@ async def finished(iens, returncode):
 
     await driver.submit(0, "sh", "-c", f"exit {actual_returncode}")
     await poll(driver, {0}, finished=finished)
+
+
+async def test_polling_bhist_fallback(not_found_bjobs):
+    driver = LsfDriver()
+    Path("mock_jobs").mkdir()
+    Path("mock_jobs/pendingtimemillis").write_text("100")
+    driver._poll_period = 0.01
+    await driver.submit(0, "sh", "-c", "sleep 1")
+    await poll(driver, {0})
```

Comment:
Given my comment [above](https://github.com/equinor/ert/pull/7299/files#r1540741183) we would be able to assert that `_poll_once_by_bhist` was called once or ?

Reply:
Hmm, I think I want to keep that as an implementation detail, this test should not care as long as `await poll` is working fine.